packages/shorebird_cli/lib/src/doctor/validators/shorebird_version_validator.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:shorebird_cli/src/commands/doctor_command.dart';
+import 'package:shorebird_cli/src/doctor/doctor_validator.dart';
+
+/// Verifies that the currently installed version of Shorebird is the latest.
+class ShorebirdVersionValidator extends DoctorValidator {
+  ShorebirdVersionValidator({required this.doctorCommand});
+
+  final DoctorCommand doctorCommand;
```

Comment:
Minor but since we only need the `isShorebirdVersionCurrent` function thoughts on injecting the method instead of the entire `DoctorCommand`?

Reply:
Seems reasonable, will update